providers/network/resources/tls.go

```diff
@@ -318,14 +318,20 @@ func (s *mqlTls) certificates(params interface{}) ([]interface{}, error) {
 	// We leverage the fact that params has to be created first, and that params
 	// causes this field to be set. If it isn't, then we cannot determine it.
 	// (TODO: use the recording data to do this async)
-	s.Certificates.State = plugin.StateIsSet | plugin.StateIsNull
-	return nil, nil
+	if len(s.Certificates.Data) <= 0 {
```

Comment:
Is there ever a case where len is < 0?

Reply:
if the site doesn't have a certificate, meaning it's http only, I guess

Reply:
But I now tested this again and it seem to still be somewhat flaky even with this change :thinking:

Reply:
Ah, no, the flakyness was because I changed to this branch but I forgot to edit the providers.yaml, it does seem to be reliable.